.github/workflows/template-janitor.yml

```
@@ -65,9 +70,33 @@ jobs:
         run: |
           rm -rf ${{ env.TEMPLATES_PATH }}
```

now.... when i think about it, maybe this one should omit the `-f` option. I think it would be nice if this one fails when the `TEMPLATES_PATH` changes without the proper change to the janitor.

Just noticed that the janitor action did not fail although the creator was renamed to renamer... that got me thinking
```suggestion
          rm -r ${{ env.TEMPLATES_PATH }}
```